src/bindgen/config_reader/condition_evaluator.cr

```diff
@@ -1,5 +1,3 @@
-require "../find_path/generic_version"
-
 alias Version = Bindgen::FindPath::GenericVersion
```

Comment:
I mean this line.  Didn't check where it was introduced, however, the `alias` is part of the global space (polluting it).  Thus, any code outside of Bindgen could write `Version` and get the `FindPath::GenericVersion`.  Could you move it into the method where it's used or so?

Reply:
This shouldn't either leak into the `Bindgen` module, which is basically bindgens global.  A developer could, erroneously, use the `Version` alias somewhere in new code by sheer luck  (It's a popular name for what it does).  Aliasing is fine as long it doesn't pollute and is kept locally.

Reply:
Alias can't be moved into the method. But I can move it to the ConfigReader module.
```diff
-alias Version = Bindgen::FindPath::GenericVersion
-
 module Bindgen
   module ConfigReader
+    alias Version = Bindgen::FindPath::GenericVersion
+
```